com.unity.cinemachine/Tests/Runtime/CmColliderTests.cs

```diff
@@ -51,35 +49,29 @@ public IEnumerator CheckSmoothingTime()
             m_Collider.DampingWhenOccluded = 0;
             var originalCamPosition = m_Vcam.State.GetFinalPosition();
 
-            yield return null; 
-            m_Brain.ManualUpdate();
+            yield return UpdateCinemachine();
+            
             Assert.That(originalCamPosition, Is.EqualTo(m_Vcam.State.GetFinalPosition()).Using(Vector3EqualityComparer.Instance));
 
             var obstacle = GameObject.CreatePrimitive(PrimitiveType.Cube);
             obstacle.transform.position = originalCamPosition; // place obstacle so that camera needs to move
             
-            yield return WaitForOnePhysicsFrame();
-            m_Brain.ManualUpdate();
+            yield return WaitForOnePhysicsFrame(); // ensure that moving the collider (obstacle) takes affect
+            yield return UpdateCinemachine();
             
             // Camera moved check
             Assert.That(originalCamPosition, Is.Not.EqualTo(m_Vcam.State.GetFinalPosition()).Using(Vector3EqualityComparer.Instance));
             Assert.That(new Vector3(0, 0, -4.4f), Is.EqualTo(m_Vcam.State.GetFinalPosition()).Using(Vector3EqualityComparer.Instance));
             
             UnityEngine.Object.Destroy(obstacle);
+
+            yield return WaitForOnePhysicsFrame(); // ensure that the obstacle's collider is removed
+            yield return WaitForSeconds(m_Collider.SmoothingTime);
+            yield return UpdateCinemachine();
             
-            // wait smoothing time and a frame so that camera move back to its original position
-            var timerStart = CinemachineCore.CurrentTime;
-            yield return WaitForOnePhysicsFrame();
-            do
-            {
-                m_Brain.ManualUpdate();
-                yield return null;
-            } while ((CinemachineCore.CurrentTime - timerStart) < m_Collider.SmoothingTime);
-            
-            m_Brain.ManualUpdate();
             Assert.That(originalCamPosition, Is.EqualTo(m_Vcam.State.GetFinalPosition()).Using(Vector3EqualityComparer.Instance));
```

Comment:
Instead of `Vector3EqualityComparer.Instance` maybe make a member equality comparer instance, with the thresholds explicitly set, and use that.  I think it could be more readable and understandable.  Right now, I don't know what the equality threshold is without hunting around.